go-get: add page (#6491)
tldr-pages/tldr: pages/common/go-get.md

# go get

> > Add dependency package, or download packages in legacy GOPATH mode.

Reviewer: Perhaps this will read better?
```suggestion
> Add a dependency package, or download packages in legacy GOPATH mode.
```

> More information: <https://pkg.go.dev/cmd/go#hdr-Add_dependencies_to_current_module_and_install_them>.

> - Add the package to go.mod in module-mode or install the packge in GOPATH-mode:

Reviewer 2: ```suggestion
- Add a specified package to go.mod in module-mode or install the package in GOPATH-mode:
```

> - Add a specified package to go.mod in module-mode or install the package in GOPATH-mode:

Reviewer 3: ```suggestion
- Add a specified package to `go.mod` in module-mode or install the package in GOPATH-mode:
```
I'd either say `the specified` or `a specific`

`go get {{example.com/pkg}}`

> - In module-mode, modify the package with a given version:

Reviewer 3: This sounds like you have to enter `module-mode` first. From what I read in the docs, what do you think about
```suggestion
- Modify the package with a given version in module-aware mode:
```
This also applies to the last example

> `go get {{example.com/pkg@v1.2.3}}`

Reviewer 4: I think this is the correct token syntax:
```suggestion
`go get {{example.com/pkg}}@{{v1.2.3}}`
```

- In module-mode, remove the package:

> `go get {{example.com/pkg@none}}`

Reviewer 4: ```suggestion
`go get {{example.com/pkg}}@{{none}}`
```

# go get

> In Module mode, add dependency package.
> More information: <https://pkg.go.dev/cmd/go#hdr-Add_dependencies_to_current_module_and_install_them>.

> - In module-mode, add package to go.mod:
> 
> `go get example.com/pkg`

Reviewer 5: ```suggestion
-  Add the package to go.mod in module-mode or install the packge in GOPATH-mode:

`go get {{example.com/pkg}}`
```

Author: ok, It does clearer

> - In module-mode, modify the packge with version:
> 
> `go get example.com/pkg@v1.2.3`

Reviewer 5: ```suggestion
- In module-mode, modify the package with a given version:

`go get {{example.com/pkg@v1.2.3}}`
```

> - In module-mode, remove the packge:
> 
> `go get example.com/pkg@none`

Reviewer 5: ```suggestion
- In module-mode, remove the package:

`go get {{example.com/pkg@none}}`
```

> In GOPATH mode, download packages into $GOPTAH/src.

> > More information: <https://pkg.go.dev/cmd/go#hdr-Legacy_GOPATH_go_get>.

Reviewer 5: Please move this description below the title and keep only one information link that is more suitable for this page.

- In GOPATH-mode, download and install the packge:

> `go get example.com/pkg`

Reviewer 5: I found this example is the same as the first one, so I give a suggestion of a combination one at the first comment.
